api-client/src/models/LocationModel.ts

```diff
@@ -113,6 +114,13 @@ class LocationModel {
     )
   }
 
+  get coordinates(): Position | null {
```

Comment:
You should also remove the other definition of this method and use this new method :)